types/index.d.ts

```diff
@@ -893,7 +893,7 @@ declare namespace Realm {
             removeAll: () => number;
         }
 
-        const MutableSubscriptions: {
+        const MutableSubscriptionSet: {
             new(): never; // This type isn't supposed to be constructed manually by end users.
             readonly prototype: SubscriptionSet;
```

Comment:
Should this be `MutableSubscriptionSet`?
```suggestion
            readonly prototype: MutableSubscriptionSet;
```

Reply:
Indeed it should